WiseUnpacker/SabreTools.Serialization.Deserializers/WiseSectionHeader.cs

```diff
@@ -87,39 +87,193 @@ public class WiseSectionHeader : BaseBinaryDeserializer<SectionHeader>
-            if (headerLength == 17)
-                return header;
+                byte[] stringBytes = data.ReadBytes(versionSize);
+                header.NonWiseVersion = Encoding.ASCII.GetString(stringBytes);
+                if (localWisOffset <= 77)
+                {
+                    header.PreFontValue = data.ReadBytes(2);
+                }
+                else
+                {
+                    header.PreFontValue = data.ReadBytes(4);
+                }
+            }
+            else // If that third byte is 0x01, no NonWiseVersion string is present.
+            {
+                header.PreFontValue = data.ReadBytes(3);
+            }
+            header.FontSize = data.ReadByte(); 
+            int preStringBytesSize = WiseSectionPreStringBytesSize[localWisOffset];
+            if (header.Version[1] == 0x01)
+            {
+                preStringBytesSize = 2; // hack for Codesited5.exe , very early and very strange.
+            }
+            header.StringValues = data.ReadBytes(preStringBytesSize);
+            List<byte> stringList = new List<byte>(); // List of string bytes to be set to final value
+            int counter = 0;
+            bool endNow = false;
+            bool languageSection = false;
+            int languageSectionCounter = 0;
+            while (counter < preStringBytesSize) // Iterate pre-string byte array
+            {
+                byte currentByte = header.StringValues[counter];
+                if (languageSectionCounter == 2) // now doing third byte after language section begins
+                {
+                    if (currentByte == 0x00) // this should never happen.
+                    {
+                        endNow = true;
+                        break;
+                    }
+                    else if (currentByte == 0x01) 
+                    {
+                        int extraLanguages = header.StringValues[counter + 1];
+                        for (int i = 0; i < extraLanguages; i++)
+                        {
+                            byte[]? incrementBytes = data.ReadBytes(2);
+                            string? extraLanguageString = data.ReadNullTerminatedAnsiString();
+                            if (extraLanguageString == null) // this should never happen
+                            {
+                                return null;
+                            }
+                            byte[]? extraLanguageStringArray = Encoding.ASCII.GetBytes(extraLanguageString);
+                            stringList.AddRange(incrementBytes);
+                            stringList.AddRange(extraLanguageStringArray);
+                        }
+                        break;
+                    }
+                }
+                else if (currentByte == 0x01) // Prepends non-string-size indicators
+                {
+                    // 01 01 01 01: entering font section
+                    // 01 5D 5C 01: link section; 5D and 5C are string sizes
```

Comment:
Is it worthwhile to define these as `public static readonly` values in `Constants`?